src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -947,13 +948,30 @@ public void onProgressChanged(long numBytes,
 	}
 
 	/**
-	 * for Web Link and citation type of resources, append ".html" to the file name String
+	 * If file extension is missing, look up the extension by file MIME type
```

Comment:
Now changed the comments to:

	 * If file extension is missing, look up the extension by file MIME type and add extension if found;
	 * If file extension is still missing, append ".html" for Web Link and citation type of resources